spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -250,13 +251,18 @@ public void onSuccess(PullResponse pullResponse) {
 	@Override
 	public List<AcknowledgeablePubsubMessage> pull(
 			String subscription, Integer maxMessages, Boolean returnImmediately) {
+		SubscriberStub subscriberStub = this.subscriberFactory.createSubscriberStub(subscription);
```

Comment:
You probably only want to create one if it's not already in the map.
I would just create a helper method like `getSubscriberStub(subscription)`.